locales/ko-KR.yml

```diff
@@ -79,6 +79,7 @@ followRequestPending: "팔로우 허가 대기중"
 enterEmoji: "이모지 입력"
 renote: "Renote"
 unrenote: "Renote 취소"
+unrenoteConfirm: "정말로 Renote를 취소하시겠습니까?"
```

Comment:
```suggestion
```

[DO NOT edit locale files except ja-JP.yml.](https://github.com/syuilo/misskey/blob/develop/locales/README.md)

It would be good to add the following to ja-JP.yml

```yaml
unrenoteConfirm: このノートを削除しますか？
````